packages/kuma-gui/src/app/data-planes/views/DataPlaneDetailView.vue

```diff
@@ -38,130 +38,132 @@
           </ul>
         </template>
 
-        <div
-          class="stack"
+        <XLayout
+          type="stack"
           data-testid="dataplane-details"
         >
-          <KCard>
-            <div
-              class="stack"
-            >
-              <XTimespan
-                :start="t('common.formats.datetime', { value: Date.parse(props.data.creationTime) })"
-                :end="t('common.formats.datetime', { value: Date.parse(props.data.modificationTime) })"
-              />
-              <div
-                class="columns"
-              >
-                <DefinitionCard>
-                  <template #title>
-                    {{ t('http.api.property.status') }}
-                  </template>
-
-                  <template #body>
-                    <div class="status-with-reason">
-                      <StatusBadge :status="props.data.status" />
-                      <DataCollection
-                        v-if="props.data.dataplaneType === 'standard'"
-                        :items="props.data.dataplane.networking.inbounds"
-                        :predicate="item => item.state !== 'Ready'"
-                        :empty="false"
-                        v-slot="{ items : unhealthyInbounds }"
-                      >
-                        <KTooltip
-                          class="reason-tooltip"
-                        >
-                          <InfoIcon
-                            :color="KUI_COLOR_BACKGROUND_NEUTRAL"
-                            :size="KUI_ICON_SIZE_30"
-                          />
-                          <template #content>
-                            <ul>
-                              <li
-                                v-for="inbound in unhealthyInbounds"
-                                :key="`${inbound.service}:${inbound.port}`"
-                              >
-                                {{ t('data-planes.routes.item.unhealthy_inbound', { service: inbound.service, port: inbound.port }) }}
-                              </li>
-                            </ul>
-                          </template>
-                        </KTooltip>
-                      </DataCollection>
-                    </div>
-                  </template>
-                </DefinitionCard>
+          <XAboutSection
+            :title="t('data-planes.routes.item.about.title')"
+            :created="t('common.formats.datetime', { value: Date.parse(props.data.creationTime) })"
+            :modified="t('common.formats.datetime', { value: Date.parse(props.data.modificationTime) })"
+          >
+            <DefinitionCard layout="horizontal">
+              <template #title>
+                {{ t('http.api.property.status') }}:
```

Comment:
Could we use something similar to thi approach for colons?

https://github.com/kumahq/kuma-gui/blob/698c2a75cd3a46f4cfb4363f8199dab77f5438b3/packages/kuma-gui/src/app/common/data-card/DataCard.vue#L59-L60

Reply:
Oh, that's nice 🙂 thanks for the pointer 👍